src/object/schema_system_catalog_install_query_spec.cpp

```diff
@@ -1172,6 +1172,7 @@ sm_define_view_stored_procedure_spec (void)
 	    "END AS [return_type], "
 	  "[sp].[arg_count] AS [arg_count], "
 	  "CASE [sp].[lang] WHEN 0 THEN 'PLCSQL' WHEN 1 THEN 'JAVA' ELSE 'UNKNOWN' END AS [lang], "
+          "CASE [sp].[directive] & 1 WHEN 0 THEN 'DEFINER' ELSE 'CURRENT_USER' END AS [authid], "
```

Comment:
If _directive != 0_, is it judged as 'DEFINER'?
It may be clearer to use the equal operation instead of the bit operator, such as "[sp].[directive] = 1".

Reply:
Sorry for not to mention about this.
The directvie column will be used for other pragma attributes. such as:
- Autonomous transaction
- function result cache
- any other hints

Reply:
Do you mean it requires a separate bit allocation for the directive as below?

autonomous transaction = 2
function result cache = 4
any other hints =8

Reply:
Exactly

Reply:
OK, I see.